File: configurator/src/main/java/com/beezassistant/configurator/models/AdminUpdate.java
package com.beezassistant.configurator.models;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name="adminupdate")
public class AdminUpdate {
	
	@Id
	@GeneratedValue(strategy=GenerationType.IDENTITY)
	private int id;
	private String heading;
	private String details;
	
	public AdminUpdate() {
		super();
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getHeading() {
		return heading;
	}

	public void setHeading(String heading) {
		this.heading = heading;
	}

	public String getDetails() {
		return details;
	}

	public void setDetails(String details) {
		this.details = details;
	}
	
}

File: botapplicationtools/programs/adminupdater/AdminUpdate.py
class AdminUpdate:

    def __init__(
            self,
            ID: int,
            heading: str,
            details: str
    ):
        self.__id = ID
        self.__heading = heading
        self.__details = details

    @property
    def getId(self):
        return self.__id

    @property
    def getHeading(self):
        return self.__heading

    @property
    def getDetails(self):
        return self.__details
